spec/models/hmpps_api/offender_spec.rb

```diff
@@ -219,24 +219,28 @@
   end
 
   describe '#over_18?' do
+    subject {
+      build(:offender, dateOfBirth: dob.to_s)
+    }
+
     context 'with a date of birth 50 years ago' do
-      before { subject.date_of_birth = 18.years.ago }
+      let(:dob) { 18.years.ago }
 
       it 'returns true' do
         expect(subject.over_18?).to eq(true)
       end
     end
 
     context 'with a date of birth just under 50 years ago' do
```

Comment:
This context block is wrong – it seems worth fixing whilst we're here:

**~just under 50 years ago~ just under 18 years ago**